layouts/partials/sidebar-tree.html

```diff
@@ -1,36 +1,35 @@
-{{/* We cache this partial for bigger sites and set the active class client side. */}}
-{{ $sidebarCacheLimit := cond (isset .Site.Params.ui "sidebar_cache_limit") .Site.Params.ui.sidebar_cache_limit 2000 -}}
-{{ $shouldDelayActive := ge (len .Site.Pages) $sidebarCacheLimit -}}
+{{/* Always cache this partial; set the active class client side. */}}
+{{ $shouldDelayActive := true }}
```

Comment:
`sidebar_cache_limit` is already set to 1 in the Hugo config, so effectively `shouldDelayActive` is true.

Why introduce a difference between the Docsy partial and this K8s override? My hope is that eventually K8s will have as few layout overrides as possible so that it can benefit from easier updates when new Docsy releases come out.